crates/uv/tests/it/run.rs

```diff
@@ -2737,6 +2737,79 @@ fn run_script_explicit_directory() -> Result<()> {
     Ok(())
 }
 
+#[test]
+#[cfg(windows)]
+fn run_gui_script_explicit() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let test_script = context.temp_dir.child("script");
+    test_script.write_str(indoc! { r#"
+        # /// script
+        # requires-python = ">=3.11"
+        # dependencies = [
+        #   "PyQt5",
+        # ]
+        # ///
+        import sys
+        import os
+        from PyQt5.QtWidgets import QApplication, QMainWindow
+        from PyQt5.QtCore import QTimer
+        
+        # Log environment info
+        print("Python executable:", sys.executable, file=sys.stderr)
+        print("Working directory:", os.getcwd(), file=sys.stderr)
+        
+        try:
+            app = QApplication(sys.argv)
+            window = QMainWindow()
+            window.setWindowTitle("UV GUI Test")
+            window.setGeometry(100, 100, 200, 100)
+            window.show()
+            
+            # Close after 1 second
+            QTimer.singleShot(1000, app.quit)
+            sys.exit(app.exec_())
+        except Exception as e:
+            print(f"Error initializing GUI: {e}", file=sys.stderr)
+            sys.exit(1)
+    "#})?;
+
+    let output = context.run().arg("--script").arg("script").output()?;
```

Comment:
Should this be using `--gui-script`?

Reply:
you are right, and I'm slightly confused why/how we passed tests with that typo... Changing now and will see how it behaves!